packages/extension-ui/src/components/ButtonWithSubtitle.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Button } from '.';
+import styled from 'styled-components';
+
+interface ButtonWithSubtitleProps {
+  title: string;
+  children: string;
+  to: string;
+}
+
+export default function ButtonWithSubtitle ({ title, children, to }: ButtonWithSubtitleProps): React.ReactElement<ButtonWithSubtitleProps> {
+  return (
+    <StyledButton to={to}>
+      <p>{title}</p>
+      <span>{children}</span>
```

Comment:
Don't really like that title is passed as a prop and subtitle as children – it's not obvious. I guess it would be better to pass both as props